docs/5g-on-helium/cbrs-radios.mdx

```diff
@@ -150,6 +143,20 @@ Neighborhoods.
 
 :::
 
+### Spectrum Access `FAILURE` Status
+
+This status is alarming, but do not let it scare you.
+
+For newly installed Small Cell Radios, this "Failure" status is likely because the registration
+process has not been completed and submitted to the SAS.
+
+You may see information similar to this:
+
+> SAS status code: 200 SAS response message: Response data: installationParam
```

Comment:
Where would they see this status code?

Reply:
this is only present during a SAS `FAILURE` state. top right corner of screen, but not always.